modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -314,6 +325,41 @@ spec:
 <1> Include the client ID.
 <2> Include the tenant ID.
 <3> Include the subscription ID.
+====
+
+* If the cluster is in {gcp-wid-short} mode, include the following fields:
++
+.Example `Subscription` object with {gcp-wid-short} variables
+[%collapsible]
+====
+[source,yaml]
+----
+kind: Subscription
+# ...
+spec:
+ config:
+   env:
+   - name: AUDIENCE
+     value: "<audience_url>" <1>
+   - name: SERVICE_ACCOUNT_EMAIL
+     value: "<service_account_email>" <2>
+----
```

Comment:
@adellape Could you help to double confirm this part? 
After the operator is installed into the cluster, the Subscription will add 4 fields, which are PROJECT_NUMBER, POOL_ID, PROVIDER_ID, and SERVICE_ACCOUNT_EMAIL 
So the Subscription YAML file will seem like below:
```
spec:
  channel: fast
  config:
    env:
      - name: PROJECT_NUMBER
        value: test
      - name: POOL_ID
        value: test
      - name: PROVIDER_ID
        value: test
      - name: SERVICE_ACCOUNT_EMAIL
        value: test
```

Reply:
cc @dlom

Reply:
Yes. I think we'd better double check on this.
The POOL_ID/PROVIDER_ID/PROJECT_NUMBER parameters are inputted on console but actually the AUDIENCE (assembled by these 3 parameters) is the one used in CredentialsRequest by CCO.Here is the [spec](https://docs.google.com/document/d/1iFNpyycby_rOY1wUew-yl3uPWlE00krTgr9XHDZOTNo/edit), but I didn't find the logic of assembling these 3 parameters to AUDIENCE in this [console PR](https://github.com/openshift/console/pull/14086) . I guess this is in the operator part of change(The operator reads the config env vars and put it into CredentialsRequest)?
@dlom @cyril-ui-developer Could you please check and confirm? thanks

Reply:
> Yes. I think we'd better double check on this. The POOL_ID/PROVIDER_ID/PROJECT_NUMBER parameters are inputted on console but actually the AUDIENCE (assembled by these 3 parameters) is the one used in CredentialsRequest by CCO.Here is the [spec](https://docs.google.com/document/d/1iFNpyycby_rOY1wUew-yl3uPWlE00krTgr9XHDZOTNo/edit), but I didn't find the logic of assembling these 3 parameters to AUDIENCE in this [console PR](https://github.com/openshift/console/pull/14086) . I guess this is in the operator part of change(The operator reads the config env vars and put it into CredentialsRequest)? @dlom @cyril-ui-developer Could you please check and confirm? thanks

cc: @jhadvig

Reply:
@jianping-shu yes, based on the [doc](https://docs.google.com/document/d/1iFNpyycby_rOY1wUew-yl3uPWlE00krTgr9XHDZOTNo/edit#heading=h.f8rp6f9yhn73) and inpit from @dlom the  logic of assembling these 3 parameters to AUDIENCE should be handled by the operator.

Reply:
Thanks for confirming this!

Reply:
Yes, that is correct